completions/bash/cou_bash_completion

```diff
@@ -20,6 +20,16 @@ _subcommand_args() {
     esac
 }
 
+_sub_subcommand_args() {
+    local cmd=$1
+    case "${cmd}" in
+        hypervisors)
+            opts="--machine --availability-zone"
+            child_commands=""
```

Comment:
This mean you will not see common options such as `--help`, `--verbose` showing for `hypervisors`, which it's not correct.